tests/integration/preprocessor/_io/test_load.py

```diff
@@ -120,6 +121,134 @@ def test_load_ncdata():
     assert not cube.coords()
 
 
+def test_load_zarr_local():
+    """Test loading a Zarr store as ncdata.NcData via Xarray."""
+    zarr_path = (
+        Path(importlib_files("tests"))
+        / "sample_data"
+        / "zarr-sample-data"
+        / "example_field_0.zarr2"
+    )
```

Comment:
You should also add a test where you try to open `str(zarr_path)` (see my comment above)